src/eq_schema/Section.js

```diff
@@ -0,0 +1,17 @@
+const Group = require("./Group");
+const { getText } = require("../utils/HTMLUtils");
+
+class Section {
+  constructor(section) {
+    this.id = `section-${section.id}`;
```

Comment:
I don't think we should be prefixing the `section-`?

Reply:
@jonnyshaw89 https://github.com/ONSdigital/eq-publisher/blob/57a4a97ec5c8ab5ffa98a885fe04f89aa28c92be/src/eq_schema/Section.js#L6

Reply:
Any reason why not? I think we do this consistently for Group, Page and Answer as well.

Reply:
From my memory, IDs needed to be globally unique, so prefixing with with type ensures this (since every table will have an a row with, say, id `1`).

Reply:
The schema validator checks that all IDs are unique across the schema. Since author has no concept of groups we need to use the same ID for both section and group. And since IDs need to be unique we need to add the type.